3-CustomWebUI/Console-Interactive-CustomWebUI/CustomWebUI/CustomBrowserWebUi.cs

```diff
@@ -0,0 +1,166 @@
+﻿using Microsoft.Identity.Client.Extensibility;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using System.Net;
+using System.Net.Sockets;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Web;
+
+namespace Console_Interactive_CustomWebUI.CustomWebUI
+{
+    internal class CustomBrowserWebUi : ICustomWebUi
```

Comment:
I'm not opposed to a sample showing this extensibility point, but not really sure what problem this solves? Our main problem is that we don't have a embedded web ui on .net core. We have it on our roadmap to publish a nuget package that exposes this using the `ICustomWebUi` implementation. I think that's the real scenario. 

Perhaps as an interim we could repurpose this sample for that? I can move the necessary files over.

Reply:
@jmprieur Did I miss understand the scenario proposed in the task?

Reply:
This is the work item

Reply:
I was only meaning mentionning the custom HTML / strings, but not necessarilly reimplementing a view from stratch. This will probably scare our customers?